Application Settings/StudentRegestrationController.cls

```diff
@@ -1,28 +1,43 @@
 public with sharing class StudentRegestrationController {
-    public Student__c studentInstance{get;set;}
-    public String languageOption{get;set;}
-    public List<SelectOption> language;
+    public Student__c studentInstance{ get; set; }
```

Comment:
Indentation should be like this -
public Student__c studentInstance {get; set;}
Also apply below